pkg/build/nodeimage/const_cni.go

```diff
@@ -95,7 +95,7 @@ spec:
       serviceAccountName: kindnet
       containers:
       - name: kindnet-cni
-        image: kindest/kindnetd:v20210220-5b7e6d01
+        image: aojea/kindnetd:0.6.0
```

Comment:
we should continue to use date-sha and you should start pushing to the real registry (please ensure fully rebased before pushing though)

Reply:
this is the main blocker for merge I think. the other stuff is more reasonable to iterate in follow ups